doc/classes/Rect2i.xml

```diff
@@ -70,41 +82,51 @@
 			<return type="Rect2i" />
 			<param index="0" name="to" type="Vector2i" />
 			<description>
-				Returns a copy of this [Rect2i] expanded so that the borders align with the given point.
+				Expands this rectangle to align the given [param to] point around the edges, if necessary.
```

Comment:
The previous wording "Returns a copy of" was better, because the new formulation could be interpreted as "expand" changes the `Rect2i`.

Reply:
Identical answer to https://github.com/godotengine/godot/pull/69816#discussion_r1279633365 (referring for cleanliness sake)